app/src/org/commcare/android/database/user/models/FormRecord.java

```diff
@@ -75,10 +75,13 @@
     public static final String STATUS_JUST_DELETED = "just-deleted";
 
     public static final String QuarantineReason_LOCAL_PROCESSING_ERROR = "local-processing-error";
+    public static final String QuarantineReason_SERVER_PROCESSING_ERROR = "server-processing-error";
     public static final String QuarantineReason_RECORD_ERROR = "record-error";
     public static final String QuarantineReason_MANUAL = "manual-quarantine";
     public static final String QuarantineReason_FILE_NOT_FOUND = "file-not-found";
 
+    private static final String QUARANTINE_REASON_AND_DETAIL_SEPARATOR = "@@SEP@@";

```

Comment:
I assume this is just to prevent the need for a migration which would add a new field?

Reply:
Yes.. There have been so many damn migrations in this class and I just couldn't bring myself to do another one for a not-very-essential String field...